packages/cli/src/cmds/beacon/options.ts

```diff
@@ -110,6 +111,13 @@ export const beaconExtraOptions: CliCommandOptions<BeaconExtraArgs> = {
     type: "number",
   },
 
+  persistOrphanedBlocksDir: {
+    description: "Enable and specify a directory to persist orphaned blocks",
```

Comment:
noticed that the flag description is not accurate as this only configures the directory, to persist orphaned blocks need to add `--chain.persistOrphanedBlocks` flag